cmd/generate/cmd_util.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nginxinc/nginx-go-crossplane/internal/generator"
+)
+
+type filterFlagReader struct {
```

Comment:
Nit: the name "reader" makes me think of `io.Reader` which this does not implement. You could just called it `filters`?

Reply:
applied: filterFlagReader -> filterFlag